.env

```diff
@@ -1,5 +1,6 @@
 # This file is commited. Do not store secrets here
 REACT_APP_ENV=staging
+#REACT_APP_ENV=development
```

Comment:
should we uncomment this? Or at least update scripts `"dev": "cross-env REACT_APP_ENV=development yarn start"`.

Reply:
Also, I just noticed that we don't have `cross-env` anymore, so it should be installed

Reply:
I'm not sure tbh. Do you think we should use `development` env by default?

Reply:
I like the `"dev": "cross-env REACT_APP_ENV=development yarn start"` bit, commenting uncommenting would be more annoying imo

Reply:
also harder to figure out for new people looking at the repo

Reply:
Let's leave it as it is for now